pkg/sentry/devices/nvproxy/version.go

```diff
@@ -346,7 +366,7 @@ func Init() {
 
 		// 535.129.03 does not exist on the main branch. It branched off the main
 		// branch at 535.113.01.
-		_ = addDriverABI(535, 129, 03, "e6dca5626a2608c6bb2a046cfcb7c1af338b9e961a7dd90ac09bb8a126ff002e", v535_113_01)
```

Comment:
Could you revert these `03` -> `0o3` type conversions? It is more readable as base 10, as driver version numbers use base 10.

Reply:
whoops. no idea where those came from. also removed the accidental formatting in L96 and L135.